doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -219,6 +219,59 @@ and ``coqtop``, unless stated otherwise:
 :-batch: Exit just after argument parsing. Available for ``coqtop`` only.
 :-verbose: Output the content of the input file as it is compiled.
   This option is available for ``coqc`` only.
+:-native-compiler (yes|no|ondemand): Enable the :tacn:`native_compute`
+  reduction machine and precompilation to ``.cmxs`` files for future uses
+  by :tacn:`native_compute`.
+  ``yes`` and ``no`` enable/disable both :tacn:`native_compute` (any call
+  to it will default back to :tacn:`vm_compute`) and precompilation.
+  Note that with ``yes``, compilation will fail if any dependency has not
+  been precompiled itself. ``ondemand`` enables :tacn:`native_compute`
+  but disables precompilation. All missing dependencies will then temporarily
+  be recompiled on the fly when caling :tacn:`native_compute`, which can be
+  very slow. Since version 8.13, the default value is set at configure time,
+  ``-config`` can be used to retrieve it.
+  All this can be summarized in the following table:
+
+  .. _native-compiler-options:
+
+  .. list-table::
+     :header-rows: 1
+
+     * - ``configure``
+       - ``coqc``
+       - outcome
+       - requirements
+     * - yes
+       - yes (default)
+       - ``.cmxs`` + temporary
+       - ``.cmxs`` of dependencies
+     * - yes
+       - no
+       - none
+       - none
+     * - yes
+       - ondemand
+       - temporary
+       - none
+     * - no
+       - yes, no, ondemand
+       - none
+       - none
+     * - ondemand
+       - yes
+       - ``.cmxs`` + temporary
+       - ``.cmxs`` of dependencies
+     * - ondemand
+       - no
+       - none
+       - none
+     * - ondemand
+       - ondemand (default)
+       - temporary
+       - none
+
+:-native-output-dir: Set the directory in which to put the aforementionned
```

Comment:
"aforementioned"

Reply:
What the... How are users even supposed to guess that this button means "insert a suggestion"?! Thanks a lot.

Reply:
If you position the cursor over the icon it shows the tooltip "Insert a suggestion".  This is a pretty standard idiom for GUIs.

Reply:
(it also took me some hovering to find that button)